cpp/environment/BondOrder.cc

```diff
@@ -99,27 +89,26 @@ void BondOrder::reduceBondOrder()
                      = m_local_bin_counts.begin();
                      local_bins != m_local_bin_counts.end(); ++local_bins)
                 {
-                    m_bin_counts.get()[sa_i((int) i, (int) j)] += (*local_bins)[sa_i((int) i, (int) j)];
+                    m_bin_counts((int) i, (int) j) += (*local_bins)((int) i, (int) j);
                 }
-                m_bo_array.get()[sa_i((int) i, (int) j)]
-                    = m_bin_counts.get()[sa_i((int) i, (int) j)] / m_sa_array.get()[sa_i((int) i, (int) j)];
+                m_bo_array((int) i, (int) j)
```

Comment:
Is it still necessary to cast to `int`?
```suggestion
                m_bo_array(i, j)
```